const a = () => 1;

function test() {
  throw new Error('1');
}

test();
